app/javascript/dashboard/assets/scss/_layout.scss

```diff
@@ -9,87 +9,25 @@ body {
 }
 
 .app-wrapper {
-  @include full-height;
-  flex-grow: 0;
-  min-height: 0;
-  width: 100%;
+  @apply h-full flex-grow-0 min-h-0 w-full;
+
+  .button--fixed-top {
+    @apply fixed ltr:right-2 rtl:left-2 top-2 flex flex-row;
+  }
```

Comment:
Ideally, for the misaligned button, we should not be using fixed positioning. Instead, it should be enclosed in the parent container with style flex, and aligned using flex alignment properties, see how it is done in the resolve-actions in the conversation view

Reply:
@scmmishra For the resolve-actions, I think it's straightforward. In this case. There are multiple files with different actions for the button. Currently, the way we have done here is there is a Wrapper component that includes a setting header and different settings route view. And all the button actions are done in different components. In that case, we need to change the logic that we did.

Reply:
Ah, understood, a solution is to solve this using portals, but that seems like an overkill. We can let this go for now then, and take a look at it at a later point. Feel free to resolve this comment

Reply:
Sure